README.md

```diff
@@ -4,18 +4,52 @@ Better Reflection
 Mimics PHP's [reflection API](http://php.net/manual/en/book.reflection.php) but without actually loading the class at
 any point. Like magic. Idea credit goes to @ocramius.
 
-Example usage:
+Example usage with the Composer autoloader:
 
 ```php
 <?php
 
 $classLoader = require "vendor/autoload.php";
 
-$reflector = new Reflector($classLoader);
-
+$reflector = new Reflector(new ComposerSourceLocator($classLoader));
 $reflectionClass = $reflector->reflect('Foo\Bar\MyClass');
 
 echo $reflectionClass->getShortName(); // MyClass
 echo $reflectionClass->getName(); // Foo\Bar\MyClass
 echo $reflectionClass->getNamespaceName(); // Foo\Bar
 ```
+
+Example usage for loading a class from a specific file:
+
+```php
+<?php
+
+$reflector = new Reflector(new FilenameSourceLocator('path/to/MyApp/MyClass.php'));
```

Comment:
As per discussion, `FilenameSourceLocator` should probably be something like `InSingleFileSourceLocator`